saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/ImageSensors.scala

```diff
@@ -0,0 +1,13 @@
+package edu.illinois.cs.cogcomp.saulexamples.mSpRL2017
+
+import edu.illinois.cs.cogcomp.saulexamples.nlp.BaseTypes.{ Image, Segment, SegmentRelation }
+
+object ImageSensors {
+
+  def image_segment_link(i: Image, s: Segment): Boolean = {
```

Comment:
please don't use underscores. The convention we have used is so far in other data models suggests names like `imageToSegment`